Issues/Week272.md

```diff
@@ -1,7 +1,7 @@
 
 **Articles**
 
-* 
+* [Create your own CocoaPods library](https://medium.com/flawless-app-stories/create-your-own-cocoapods-library-da589d5cd270), by [@jeantimex](https://twitter.com/jeantimex)
 
```

Comment:
[Open Source user consent SDK for iOS & Android apps](https://www.smartlook.com/blog/open-source-user-consent-sdk-ios-android-apps-smartlook/), by [@getSmartlook](https://twitter.com/getSmartlook)